c/kore/hello/src/hello.c

```diff
@@ -0,0 +1,43 @@
+#include <kore/kore.h>
+#include <kore/http.h>
+
+int		home(struct http_request *);
+int		user(struct http_request *);
+int		user_details(struct http_request *);
+
+int
+home(struct http_request *req)
+{
+	http_response(req, 200, NULL, 0);
+	return (KORE_RESULT_OK);
+}
+
+int
+user(struct http_request *req)
+{
+	http_response(req, 200, NULL, 0);
+	return (KORE_RESULT_OK);
+}
+
+int
+user_details(struct http_request *req)
+{
+	char		*id;
+	size_t		len;
+	struct kore_buf		*buf;
+	u_int8_t		*data;
+
+	http_populate_get(req);
+
+	buf = kore_buf_alloc(1024);
```

Comment:
Don't allocate HEAP in a callback, only allocate on the STACK unless it is significantly larger, which is not the case here.